tests/edge-cases.spec.ts

```diff
@@ -20,3 +20,10 @@ test("removing an autoPopover from document doesn't crash page", async ({
   ).toBeUndefined();
   await expect(popover2).toBeVisible();
 });
+
+test('styles in layers are winning over polyfilled styles', async ({
+  page,
+}) => {
+  const popover = (await page.locator('#popover13')).nth(0);
+  await expect(popover).toHaveCSS('background-color', 'rgb(208, 13, 30)');
+});
```

Comment:
Nice test :+1:

Reply:
Per this [blog post](https://playwrightsolutions.com/what-the-hex-or-how-i-check-colors-with-playwright), Playwright does expect it to be rgb. I'm also seeing `rgb()` in Playwright's own tests, but I'm not seeing this documented.

Reply:
I'm pretty sure this must be `getComputedStyle()`, so the browser is doing it.

Reply:
Looks like it is spec'd that way for `getComputedStyle()`, so this is probably okay. https://stackoverflow.com/a/67006298/5628